models/Document/Editable/Video.php

```diff
@@ -707,11 +707,11 @@ private function getDailymotionCode()
 
             $additional_params = '';
 
-            $clipConfig = is_array($config['config']['clip']) ? $config['config']['clip'] : [];
+            $clipConfig = isset($config['config']['clip']) && is_array($config['config']['clip']) ? $config['config']['clip'] : [];
```

Comment:
@BabovicT could you please fix Vimeo as well in `getVimeoCode()` then we can close the PR https://github.com/pimcore/pimcore/pull/10182/files in favor of this...thanks!